docs/bibliography.bib

```diff
@@ -177,6 +177,29 @@ @article{hellinger:2011
    number = {A9},
    doi = {10.1029/2011ja016674}
 }
+@article{johnson:2023a
+  author = {E. Johnson},
+  title = {Anterograde Collisional Analysis of Solar Wind Ion},
```

Comment:
Nice that you got to use the word "anterograde" in your title!  I'm going to assume it refers to "directed forward in time" rather than "of or denoting a type of amnesia involving inability to remember any new information". 🤔

Reply:
Yes, it refers to the direction in which the collisional analysis is performed